app/main/views/questionnaire.py

```diff
@@ -30,11 +36,11 @@
 @questionnaire_blueprint.before_request
 @login_required
 def check_survey_state():
-    json, schema = get_schema()
-    g.questionnaire_manager = QuestionnaireManager(schema, json=json)
+    g.schema_json, g.schema = get_schema()
```

Comment:
For info, in the Alpha when using Cassandra the connection pool was setup once at startup; this was done outside of any Flask mechanism as it turned out the request and application contexts in Flask are all per-request so aren't cached between requests. There may be a more 'Flask' way of doing it however, the PR related to this is here: https://github.com/ONSdigital/alpha-eq-survey-runner/pull/54